app/models/thread.php

```diff
@@ -37,19 +41,18 @@ public static function get($id)
         return new self($row);
     }
 
-    public function write(Comment $comment)
+    public function write($comment, $username, $id)
     {
         if (!$comment->validate()) {
```

Comment:
Sorry :sweat: Please put the validation to the model (of comment) instead po :bow: